app/controllers/application_pets_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ApplicationPetsController < ApplicationController
+  def update
+    application = Application.find(params[:id])
+    selected_app_pet = ApplicationPet.find(params[:pet_id])
+    if params[:approved?] == "Yes"
```

Comment:
Refactor idea to consider: Your if statement could just check for the presence of a `:approved` param with `params[:approved].present?` and then it could pass the value of the param down to a helper method or model method to handle updating the AP status and adoptability based on the value of that param. That could clean up this controller action a bit and consolidate logic

Also, just to avoid issues with URL syntax, query parameters shouldn't include a `?` because that is the path symbol to indicate the beginning of query parameters. We generally only add `?` to boolean methods.